Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -71,7 +71,7 @@
                 // ---- The number of worker jobs, 0 means use the number of Logical Cores
                 "Jobs": {
                     "minJobs": 1,
-                    "maxJobs": 0
+                    "maxJobs": 8
```

Comment:
This can be checked.
As a matter of fact, the way the Settings Registry works is users shouldn't modify the `<engine-root>/Registry` files if they want to override specific settings for their machine.
Instead they should add a setreg file to the per user directories that O3DE scans for setreg files: https://www.docs.o3de.org/docs/user-guide/settings/developer-documentation/#search-locations

Instead what you can do is to create a setreg file within your `~/.o3de/Registry` if you want to override the AP maxJobs for all running AP instances regardless of project.
Or if you only want to override it for a single project, create a setreg file within the `<project-path>/user/Registry` directory